x-pack/plugins/enterprise_search/common/types/pipelines.ts

```diff
@@ -80,6 +80,7 @@ export interface CreateMlInferencePipelineParameters {
   model_id: string;
   pipeline_name: string;
   source_field: string;
+  field_mappings?: FieldMapping[];
```

Comment:
Currently this property is only used for populating the attachable pipelines dropdown list.